Assets/MRTK/Core/Utilities/SceneContent/MixedRealitySceneContent.cs

```diff
@@ -73,23 +69,26 @@ public void InitializeSceneContent()
                 return;
             }
 
+            MixedRealityExperienceSettingsProfile experienceSettingsProfile = MixedRealityToolkit.Instance.ActiveProfile.ExperienceSettingsProfile;
+
             if (alignmentType == AlignmentType.AlignWithExperienceScale)
             {
                 bool experienceAdjustedByXRDevice =
-#if UNITY_2019_3_OR_NEWER
+#if UNITY_2020_1_OR_NEWER
                     XRSubsystemHelpers.InputSubsystem != null && !XRSubsystemHelpers.InputSubsystem.GetTrackingOriginMode().HasFlag(TrackingOriginModeFlags.Unknown);
 #else
-                    XRDevice.isPresent && XRDevice.GetTrackingSpaceType() == TrackingSpaceType.RoomScale;
-#endif // UNITY_2019_3_OR_NEWER
+                (XRSubsystemHelpers.InputSubsystem != null && !XRSubsystemHelpers.InputSubsystem.GetTrackingOriginMode().HasFlag(TrackingOriginModeFlags.Unknown)) ||
```

Comment:
Since the XRDevice check is checking for `TrackingSpaceType.RoomScale`, should this not instead check for the current mode to be `TrackingOriginModeFlags.Floor`?

Reply:
since this bool is called `experienceAdjustedByXRDevice`, but `TrackingOriginModeFlags.Device` will be head-origin which I don't think adjusts the experience in the way this is trying to account for

Reply:
Yeah, I'll fix the bool check to match what we have in XRSDKBoundarySystem
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/265a0ea3621c84b88e4b02e7fcd1b5682afef56a/Assets/MRTK/Providers/XRSDK/XRSDKBoundarySystem.cs#L54-L58